statsUsageWords.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+TEXT=$(cat "$1" | tr '[:upper:]' '[:lower:]')
+
+declare -A WORDS
+
+for WORD in $TEXT
+do
+    CHARACTER_COUNT=$(echo -n $WORD | wc -m)
```

Comment:
I would check that the words do not end in a period or a comma before processing them.